### ISSUE TITLE
will default shuffle the images during traing?

### ISSUE THREAD

**Issue by alexsun009** [WongKinYiu/ScaledYOLOv4#53]
Hi, I check the code but seems it does not shuffle the image list during traing; may I know is it true or what should I do if I want to shuffle during traing? thanks

**Comment by WongKinYiu** [comment]
yes, there are no shuffle in default, but [mosaic](https://github.com/WongKinYiu/ScaledYOLOv4/blob/yolov4-csp/utils/datasets.py#L610) data augmentation will random pick 3 additional images.

**Comment by liminghu** [comment]
Can I turn off mosaic?
I tried 
mixup: 0.0  # image mixup (probability)
mosaic: 0.0  # image mosaic (probability) 

I checked the images at runs, the mosaic augmentation is still enabled.

**Comment by liminghu** [comment]
I figured it out.
it is a parameter: --rect
